system/xmllib/xpathprocessor.hpp

```diff
@@ -152,11 +153,11 @@ extern "C" XMLLIB_API IXpathContext*  getXpathContext(const char * xmldoc, bool
 #define ESDLScriptCtxSection_ScriptRequest "script_request"
 #define ESDLScriptCtxSection_ScriptResponse "script_response"
 
-interface IEsdlScriptContext : extends IInterface
+interface IXmlDocContext : extends IInterface
```

Comment:
I'm not sure what the right name is, but I think IXmlDocContext is a bit too generic.  We're dealing with sections, cleaning up between scripts, directing tracing.